src/javascript/app/base/header.js

```diff
@@ -313,8 +313,12 @@ const Header = (() => {
                         result = verification_length && document.status === 'expired';
                         break;
                     }
+                    case 'expired_income': {
+                        result = verification_length && income.status === 'expired';
+                        break;
```

Comment:
could you please check it again.
as I know income can have only following statuses: 'none', 'verified', 'rejected', 'locked', 'pending'